cpanfile

```diff
@@ -78,6 +78,7 @@ on 'develop' => sub {
   requires "Test::Needs" => "0";
   requires "Test::Pod" => "1.41";
   requires "Test::Pod::Coverage" => "1.08";
+  requires "Test::Pod::LinkCheck" => "0";
```

Comment:
I think this needs to be `develop_requires` as we don't need to run this outside if the `dzil` build.